jsx/Help.js

```diff
@@ -0,0 +1,151 @@
+import RMarkdown from 'jsx/Markdown';
+import React, {useState, useEffect} from 'react';
+
+/**
+ * Help Widget
+ *
+ * Display help content in a tooltip
+ *
+ * @param props
+ * @author LORIS Team
+ * @version 1.0.0
+ */
+
+/**
+ * Help
+ *
+ * @param {object} props - React props
+ * @return {JSX}
+ */
+const Help = (props) => {
+  const [isActive, setIsActive] = useState(props.content ? true : false);
+  const [format, setFormat] = useState(null);
+  const [content, setContent] = useState(null);
+  const [isMounted, setIsMounted] = useState(true);
+
+  const getParams = {
+    testName: loris.TestName,
+    ...(loris.Subtest !== '' && {subtest: loris.Subtest}),
+  };
+
+  useEffect(() => {
+    props.content && setContent(props.content);
+  }, [props.content]);
+
+  useEffect(() => {
+    !props.content && fetch(
+      loris.BaseURL
+      + '/help_editor/ajax/help.php?'
+      + new URLSearchParams(getParams),
+      {method: 'GET'},
+    ).then((response) => {
+      if (!response.ok) {
+        console.error(response.status);
+        return;
+      }
+      response.json().then(
+        (data) => {
+          if (!isMounted) return;
+          setFormat(data.format);
+
+          // Render Markdown in wrap div.
+          // If help content is from Markdown helpfile.
+          if (format === 'markdown') {
+            setContent(<RMarkdown content={data.content}/>);
+          } else {
+            // If help content is from DB.
+            setContent(
+              <>
```

Comment:
I don't see why this is a fragment instead of encapsulated in a div

Reply:
Because we don't need the extra div in the DOM. Is there any issue in using a fragment?

Reply:
It's a code smell that implies that the component isn't properly encapsulated.. and there's no cost to divs.

Reply:
I'm not sure I agree that using fragments makes the component not properly encapsulated. And there is a cost in using DIV when they are not required since it complexifies the DOM tree.
https://www.geeksforgeeks.org/why-are-fragments-better-than-container-divs/

Reply:
Ignoring the assertions about performance with no benchmarks to back them up (and the fact we're talking about a single div here) in that blog post, the only thing left is "because of flexbox and grid" which is exactly wrong. If a component is encapsulated in a single tag, you can use it as an element in a flexbox or grid. If it's not, you're adding multiple elements to the parent which will affect the spacing and alignment of all the other elements in the parent as if you were adding multiple elements, not a single component.